### ISSUE TITLE
Startup crash in TCon integration when prismarine additions disabled

### ISSUE THREAD

**Issue by aran-gilman** [MinecraftModDevelopmentMods/BaseMetals#455]
Minecraft version: 1.12.2
MMDLib version: 1.0.0 rc2.36
Base Metals version: 2.5.0- c2.332

Disabling prismarine additions in BaseMetals.cfg leads to a NullPointerException in the TCon integration code. I'm guessing this was fixed by #436, but that landed 3 days after the last release available on Curseforge.

Crash Report:
[crash-2019-08-31_09.42.12-client.txt](https://github.com/MinecraftModDevelopmentMods/BaseMetals/files/3562646/crash-2019-08-31_09.42.12-client.txt)

**Comment by dshadowwolf** [comment]
No, this is still occurring and has also been reported elsewhere. I believe @YaibaToKen has been working on a fix while I've been stuck with paying work.
